tools/recetox_msfinder/macros.xml

```diff
@@ -0,0 +1,78 @@
+<macros>
+    <token name="@TOOL_VERSION@">0.1.0</token>
```

Comment:
I suppose the wrapper version should reflect the tool version, similarly to docker container.

```suggestion
    <token name="@TOOL_VERSION@">3.5.2</token>
```